BUILDING.md

```diff
@@ -73,10 +73,12 @@ sudo apt install build-essential cmake zlib1g-dev libgl1-mesa-dev libdrm-dev gcc
 
 #### Fedora / CentOS
 
+Note: This sould work on RHEL/CentOS 8 or later and recent versions of Fedora. To build on CentOS 7 or order, see [Linux with outdated Qt](#linux-with-outdated-qt).
```

Comment:
s/sould/should/
s/order/older/

Else thank you for the updates! (I have no more CentOS system.)

Reply:
Fixed. Sorry about the typos!